hub/constants.py

```diff
@@ -10,6 +10,10 @@
 MIN_FIRST_CACHE_SIZE = 32 * MB
 MIN_SECOND_CACHE_SIZE = 160 * MB
 
+# without MB multiplication, meant for the Dataset API that takes cache size in MBs
```

Comment:
why not MB multiplication here? i don't understand what these values represent without this